controllers/userController.js

```diff
@@ -0,0 +1,21 @@
+const User = require('../models/userModel');
+
+exports.getAllUsers = async (req, res) => {
+  const users = await User.find();
+
+  res.status(200).json({
+    status: 'success',
```

Comment:
200 code already states that it is success, no need to add "status: 'success' property. Also, clean restapi for /GET endpoint should return array of items